assets/src/edit-story/components/checklistCard/checklistCard.js

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+
+import { Headline, THEME_CONSTANTS } from '../../../design-system';
+import { getGridVariant } from './helpers';
+import {
+  CARD_TYPE,
+  MAX_THUMBNAILS_DISPLAYED,
+  DEFAULT_OVERFLOW_LABEL,
+} from './constants';
+
+import {
+  Wrapper,
+  Container,
+  Title,
+  Cta,
+  ThumbnailWrapper,
+  StyledOverflowThumbnail,
+  Footer,
+} from './styles';
+
+/**
+ *
+ * @param {Object} props Component props.
+ * @param {string} props.cardType the type of card  getting rendered. Selected from constants > CARD_TYPE, defaults to SINGLE_ISSUE.
+ * @param {string}  props.title text to display as title of card.
+ * @param {Object}  props.titleProps if an object is passed in it should have an onClick, these are so that issues can have buttons as titles.
+ * @param {Node} props.footer will  be rendered in the footer section of a card.
+ * @param {Node} props.cta will be rendered as the cta section of a card.
+ * @param {string} props.overflowLabel will be used as the aria label for the overflow thumbnail
+ * @param {Node}  props.thumbnail will be rendered in the thumbnail section of a card.
+ * @param  {number} props.thumbnailCount count of how  many thumbnails are getting rendered to use to manipulate grid. Defaults to 0.
+ * @return {Node} card to  display.
+ */
+const ChecklistCard = ({
+  cardType = CARD_TYPE.SINGLE_ISSUE,
+  title,
+  titleProps,
+  footer,
+  cta,
+  overflowLabel = DEFAULT_OVERFLOW_LABEL,
+  thumbnail,
+  thumbnailCount = 0,
+}) => {
+  const gridVariant = getGridVariant({
+    cardType,
+    thumbnailCount,
+    hasCta: Boolean(cta),
+  });
+
+  const hasOverflowThumbnail = thumbnailCount > 0 && thumbnailCount > 4;
+
+  return (
+    <Wrapper>
+      <Container gridVariant={gridVariant}>
+        <Title as={titleProps?.onClick ? 'button' : 'div'} {...titleProps}>
+          <Headline
+            size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.XXX_SMALL}
+            as="h4"
+          >
+            {title}
+          </Headline>
+        </Title>
+        <ThumbnailWrapper
+          $isMultiple={thumbnailCount > 1}
+          $colCount={hasOverflowThumbnail ? MAX_THUMBNAILS_DISPLAYED : 4}
+        >
+          {thumbnail}
+        </ThumbnailWrapper>
+        {hasOverflowThumbnail && (
+          <StyledOverflowThumbnail
```

Comment:
This is something I'm still noodling but it doesn't seem worth holding up review for because it'd be great to have this structure in place as the PPC issues get transferred to the new structure. Open to suggestions. To me, this is not a button because there's no action to take, it's just a placeholder that says there's content hidden. However, if you're using a screen reader we need this to be focusable so that they get a useful label and know there's more to fix than just 3 items. 

So for the time being, I have this as a button 😞 and am just passing in an aria-label, no click events.